github/repos.go

```diff
@@ -284,6 +287,7 @@ func (s *RepositoriesService) Get(owner, repo string) (*Repository, *Response, e
 	// TODO: remove custom Accept header when the license support fully launches
 	// https://developer.github.com/v3/licenses/#get-a-repositorys-license
 	req.Header.Set("Accept", mediaTypeLicensesPreview)
+	req.Header.Add("Accept", mediaTypeSquashPreview)
```

Comment:
It turns out that my hunch for using comma-separated values was correct:
https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2

From GitHub support:
> Example:
> Accept: application/vnd.github.mockingbird-preview, application/vnd.github.polaris-preview

Of course, this will need to be tested.